packages/ajna-contracts/test/pool.dpm.test.ts

```diff
@@ -680,17 +683,84 @@ describe.only("AjnaProxyActions", function () {
       const balancesQuoteAfter = {
         borrower: await usdc.balanceOf(borrower.address),
         pool: await usdc.balanceOf(poolContract.address),
+        proxy: await usdc.balanceOf(borrowerProxy.address),
       };
       const balancesCollateralAfter = {
         borrower: await wbtc.balanceOf(borrower.address),
         pool: await wbtc.balanceOf(poolContract.address),
+        proxy: await wbtc.balanceOf(borrowerProxy.address),
       };
 
       expect(balancesQuoteAfter.borrower).to.be.lt(balancesQuoteBefore.borrower);
       expect(balancesQuoteBefore.borrower).to.be.eq(balancesQuoteAfterDraw.borrower.sub(bn.six.THOUSAND.mul(2)));
       expect(balancesCollateralAfter.borrower).to.be.equal(balancesCollateralBefore.borrower);
+      expect(balancesQuoteAfter.proxy).to.be.eq(ZERO);
+      expect(balancesCollateralAfter.proxy).to.be.eq(ZERO);
     });
+    it.only("should openAndDraw and repayDebtAndWithdrawCollateral", async () => {
+      const { wbtc, borrowerProxy, poolContract, ajnaProxyActionsContract, borrower, usdc, poolInfoContract } =
+        await loadFixture(deploy);
+      const balancesQuoteBefore = {
+        borrower: await usdc.balanceOf(borrower.address),
+        pool: await usdc.balanceOf(poolContract.address),
+      };
+      const balancesCollateralBefore = {
+        borrower: await wbtc.balanceOf(borrower.address),
+        pool: await wbtc.balanceOf(poolContract.address),
+      };
+      const price = bn.eighteen.TEST_PRICE_3;
+      await depositAndDrawDebt(
+        ajnaProxyActionsContract,
+        poolContract,
+        price,
+        wbtc,
+        borrower,
+        borrowerProxy,
+        bn.six.THOUSAND,
+        bn.eight.TEN
+      );
+      await depositAndDrawDebt(
```

Comment:
The call to `depositAndDrawDebt` is duplicated. Is it on purpose? If so, can you add a comment explaining why?

Reply:
no idea. removing it